src/test/java/org/apache/maven/plugins/pmd/stubs/CustomConfigurationMavenProjectStub.java

```diff
@@ -40,35 +41,26 @@ public class CustomConfigurationMavenProjectStub extends PmdProjectStub {
 
     private List<ReportPlugin> reportPlugins = new ArrayList<>();
 
-    public CustomConfigurationMavenProjectStub() {
+    public CustomConfigurationMavenProjectStub() throws IOException, XmlPullParserException {
         MavenXpp3Reader pomReader = new MavenXpp3Reader();
-        Model model = null;
 
-        try (InputStream is = new FileInputStream(new File(getBasedir() + "/" + getPOM()))) {
-            model = pomReader.read(is);
-            setModel(model);
-        } catch (Exception e) {
-        }
+        InputStream in = new FileInputStream(getBasedir() + "/" + getPOM());
```

Comment:
stream should be closed also in tests

Reply:
OK, maybe that's not where it is, but two tests broke for that inobvious reason when I initially set this up with try with resources. I had to drop the closing to make them pass.

Reply:
If I auto-close this renamed `in` here, with try-with-resources, no test fails on me:
```
Tests run: 50, Failures: 0, Errors: 0, Skipped: 0
```
How to see the failure _when the `pomReader` closes the inputstream_? In which test? When `pomReader` does that?